src/renderer/containers/Table/CustomCells/StatusCell/index.tsx

```diff
@@ -60,12 +60,13 @@ export default function StatusCell(props: CellProps<UploadSummaryTableRow>) {
         <CheckCircleFilled className={styles.success} />
       );
     } else {
-      // Switch the incomplete status tooltip depending on if the post upload
-      // has yet to run vs if it has failed
-      if (etlProcess?.status === JSSJobStatus.FAILED) {
-        tooltip = `${tooltip} - File has been successfully uploaded to FMS, but may not be viewable in the FMS File Explorer. Attempt to make it visible in the FMS Explorer resulted in the following error: ${etlProcess.status_detail}`;
+      if (
+        props.row.original.serviceFields?.postUploadProcessing?.etl?.status ===
+        JSSJobStatus.FAILED
+      ) {
+        tooltip = `${tooltip} - File has been successfully uploaded to FMS, but may not be viewable in the File Upload App. Attempt to make it visible in the FMS Explorer resulted in the following error: ${props.row.original.serviceFields?.postUploadProcessing?.etl?.status_detail}`;
```

Comment:
I am suspicious of the changes from `etlProcess?.status` to `props.row.original.serviceFields?.postUploadProcessing?.etl?.status` on lines 64 and 67. In the [original PR](https://github.com/aics-int/aics-file-upload-app/pull/21/files#diff-739a3322a5484e0db015222c5f4f4b972756685a891a911024447a2cedb3bac6) for these changes, the change was simply replacing "FMS File Explorer" with "File Upload App". So I suspect that `etlProcess?.status` is the newer/cleaner option